apps/api/src/bans/bans.controller.ts

```diff
@@ -14,8 +25,25 @@ export class BansController {
   constructor(
     private readonly bansService: BansService,
     private readonly eventEmitter: EventEmitter2,
+    private readonly configsService: ConfigsService,
   ) {}
 
+  @Post()
+  @ApiOperation({ summary: "Create a new ban" })
+  @ApiResponse({ type: BanSchema })
+  async create(@Body() banCreationDto: BanCreationDto): Promise<BanSchema> {
+    const config = await this.configsService.findOne(banCreationDto.configId);
+    if (!config) {
+      throw new Error("Config not found");
+    }
+    // create a new ban
+    return await this.bansService.create({
```

Comment:
Do you think it's possible to add a parameter like `isManual` to differentiate manual and non manual bans ?

Reply:
This makes sense, I will implement it!